### PR TITLE
Class inheritance

### PR DESCRIPTION
**Pull request by ezraermy**
This pull request creates 'Student' and 'Teacher' classes that are inherited from 'Person' classes.
The implementation includes the following changes:
- Installed the required packages: Ruby and Linters
- Created three separate ruby files for each class
- The Student class includes a classroom attribute and a play_hooky method, while the Teacher class includes a specialization attribute and overrides the can_use_services? 

### Other requirements
- No linter errors
- Follow github flow
- Document professional way.

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+class Student < Person
+  def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
+    super(id, age, name, parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯\\(ツ)/¯'
+  end
```
Comment (cynthiainga) [inline student.rb]: - [ ] The `play_hooky` method should return `'¯\(ツ)/¯'` and not  '¯\\(ツ)/¯', please kindly fix that 🙏🏾

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```
Comment (cynthiainga) [inline person.rb]: - [OPTIONAL] I think it would be a good idea if the id can be generated randomly, it can be similar to 👉🏾  `@id = Random.rand(1..1000)`

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @ezraermy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 2 above)
Comment (ezraermy) [reply person.rb]: Hi @cynthiainga. Thank you very much for the feedback and tips. For the first one, I think you mean I used two back-slashes \\, which I should correct.

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @ezraermy , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implementing the requested changes. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**